### PR TITLE
[Shipa-763] Make shipa default resource names consistent

### PR DESCRIPTION
**Pull request by itsksaurabh**
# Description

Fixes [SHIPA-763](https://shipaio.atlassian.net/browse/SHIPA-763)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Chore (documentation addition or typo, file relocation)

## Final Checklist:

- [x] I followed standard [GitHub flow](https://guides.github.com/introduction/flow/) guidelines
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

## Additional Information (omit if empty)

Please include anything else relevant to this PR that may be useful to know.

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -11,6 +11,10 @@ shipaApi:
   image: shipasoftware/api:helm-v1.0.89
   pullPolicy: Always
   debug: false
+  # platformImageTag is the default tag for supported Shipa platform images 
+  # Any Dockerfile provided by the user having base image from shipa registry
+  # should have the supported platform image tag for compatibility with Shipa API.
+  platformImageTag: v1.2
```
Comment (aleksej-paschenko) [inline values.yaml:17]: that is from another commit

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Description
+
```
Comment (aleksej-paschenko) [inline .github/pull_request_template.md:2]: from another commit

File: scripts/common.sh [hunk 3]
```diff
@@ -9,9 +9,14 @@ wait_public_ip() {
       echo $ClusterIP
     else
       external_ip=""
-      while [ -z $external_ip ]; do
-          external_ip=$(kubectl get svc $SERVICE_NAME --template="{{range .status.loadBalancer.ingress}}{{.ip}}{{end}}")
+      while [ -z "$external_ip" ]; do
           sleep 2
+          external_ip=$(kubectl get svc $SERVICE_NAME --template="{{range .status.loadBalancer.ingress}}{{.ip}}{{end}}")
```
Comment (aleksej-paschenko) [inline scripts/common.sh:14]: please remove this change from the PR

**Review by aleksej-paschenko — CHANGES_REQUESTED** [review]
there is "shipaCore" section in values.yaml to configure shipa-core cluster
should we rename it to "shipaCluster"?

**Comment by itsksaurabh** [comment]
All those extra commits came from rebasing the branch. The branch was initially created using master but rebased it to dev after your suggestion yesterday. fixing it now.

File: templates/shipa-api-configmap.yaml [hunk 4]
```diff
@@ -26,8 +28,8 @@ data:
       user-registration: true
     provisioner: kubernetes
     platforms:
-      platformImageTag: {{ .Values.platforms.defaultPlatformImageTag }}
-      build-pool: theonepool
+      build-pool: shipa-pool
+      platformImageTag: {{ .Values.shipaApi.platformImageTag }}
```
Comment (aleksej-paschenko) [inline templates/shipa-api-configmap.yaml:32]: `Values.platforms.defaultPlatformImageTag`?

File: scripts/init-job.sh [hunk 5]
```diff
@@ -13,18 +13,17 @@ $SHIPA_CLIENT login << EOF
...
 
 TOKEN=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)
 CACERT="/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
 ADDR=$KUBERNETES_SERVICE_HOST:$KUBERNETES_SERVICE_PORT
 
 sleep 10
-
 if [[ -z $ISTIO_INGRESS_IP ]]; then
-  $SHIPA_CLIENT cluster-add shipa-core --pool=theonepool \
+  $SHIPA_CLIENT cluster-add shipa-cluster --pool=shipa-pool \
```
Comment (aleksej-paschenko) [inline scripts/init-job.sh:26]: we are renaming `shipa-core`  to `shipa-cluster` and there is `shipaCore` section in values.yaml. Should we rename it too for consistency?

File: templates/shipa-org-configmap.yaml [hunk 6]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "shipa.fullname" . }}-org-config
```
Comment (aleksej-paschenko) [inline templates/shipa-org-configmap.yaml:4]: could we have a more suitable name? it contains only org id now but  we will definitely add other values

**Review by aleksej-paschenko — CHANGES_REQUESTED** [review]

File: templates/shipa-api-configmap.yaml (hunk 4 above)
Comment (itsksaurabh) [reply templates/shipa-api-configmap.yaml:32]: I interchanged their position that is why It is appearing here. not coming from any other commit.

File: templates/shipa-org-configmap.yaml (hunk 6 above)
Comment (itsksaurabh) [reply templates/shipa-org-configmap.yaml:4]: how about `{{ template "shipa.fullname" . }}-default-organization-config` ?

File: templates/shipa-org-configmap.yaml [hunk 7]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "shipa.fullname" . }}-default-organization-config
```
Comment (aleksej-paschenko) [inline templates/shipa-org-configmap.yaml:4]: let's have a more general name for this config, currently, there is  an organization, but later we will add other things there

File: templates/shipa-api-configmap.yaml (hunk 4 above)
Comment (aleksej-paschenko) [reply templates/shipa-api-configmap.yaml:32]: and you changed the names, check again please

**Review by aleksej-paschenko — CHANGES_REQUESTED** [review]

File: templates/shipa-org-configmap.yaml (hunk 7 above)
Comment (itsksaurabh) [reply templates/shipa-org-configmap.yaml]: changed to `{{ template "shipa.fullname" . }}-defaults-configmap` as discussed

**Review by koncar — APPROVED** [review]

**Review by aleksej-paschenko — APPROVED** [review]

**Pull request merged**